### PR TITLE
Add Block Is Passable Condition

### PR DESCRIPTION
**Pull request by Mr-Darth**
### Description
Added a condition to check whether a block is passable.

---
**Target Minecraft Versions:** 1.13+
**Requirements:** 1.13+
**Related Issues:** none

### CODE REVIEW

**Review by APickledWalrus — APPROVED** [review]
LGTM

File: src/main/java/ch/njol/skript/conditions/CondIsPassable.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import ch.njol.skript.doc.Since;
+
+@Name("Is Passable")
+@Description({"Checks whether a block is passable.",
+			"A block is passable if it has no colliding parts that would prevent players from moving through it.",
+			"Blocks like tall grass, flowers, signs, etc. are passable, but open doors, fence gates, trap doors, etc. are not because they still have parts that can be collided with."
+})
+@Examples("if player's targeted block is passable")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13+")
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/conditions/CondIsPassable.java:39]: This should actually be changed to 1.13.2+. I missed that in my first look.
```suggestion
@RequiredPlugins("1.13.2+")
```

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/conditions/CondIsPassable.java (hunk 1 above)
Comment (Mr-Darth) [reply src/main/java/ch/njol/skript/conditions/CondIsPassable.java]: Done!

**Review by APickledWalrus — APPROVED** [review]
LGTM

File: src/main/java/ch/njol/skript/conditions/CondIsPassable.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsPassable.java:18]: bensku has updated the last line of the license heard which can be found [**HERE**](https://github.com/SkriptLang/Skript/blob/master/licenseheader.txt) to update accordingly

File: src/main/java/ch/njol/skript/conditions/CondIsPassable.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import ch.njol.skript.doc.Since;
+
+@Name("Is Passable")
+@Description({"Checks whether a block is passable.",
+			"A block is passable if it has no colliding parts that would prevent players from moving through it.",
+			"Blocks like tall grass, flowers, signs, etc. are passable, but open doors, fence gates, trap doors, etc. are not because they still have parts that can be collided with."
+})
+@Examples("if player's targeted block is passable")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13.2+")
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsPassable.java:39]: To keep consistency with the rest of the project I think this is more suiting
```suggestion
@RequiredPlugins("Minecraft 1.13.2+")
```

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
Just a few small things, other than that, looks good to me.

**Review by ShaneBeee — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by ShaneBeee** [comment]
Thank you for your contribution

**Pull request merged**